### PR TITLE
Fix  #176472 www.mmorpg.com

### LINKED ISSUE #176472: www.mmorpg.com
### Issue URL (Annoyance)
 [https://www.mmorpg.com/reviews/final-fantasy-vii-rebirth-review-2000130565](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.mmorpg.com%2Freviews%2Ffinal-fantasy-vii-rebirth-review-2000130565)
Username: `@rakleed`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.16.0 (4542)
Browser: | Chrome
License type: | paid
AdGuard driver: | WFP
Stealth mode options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Hide your Referrer from third-parties (www.mmorpg.com), <br>Hide your User-Agent, <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (180), <br>Disable cache for third-party requests, <br>Block trackers, <br>Disable WAP Push Message Routing Service, <br>Disable Windows Defender automatic samples submission, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js<br>https://update.greasyfork.org/scripts/433360/Nova%20YouTube.user.js<br>https://update.greasyfork.org/scripts/15222/IMDb%20Tomatoes.user.js
Referrer: | [https://opencritic.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fopencritic.com%2F)

### PR DESCRIPTION
**Pull request by vankos**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

## What problem does the pull request fix?
Hides play now sidebar and empty leftover ad space

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?

https://github.com/AdguardTeam/AdguardFilters/issues/176472

### Add your comment and screenshots

Screenshots

<details>

<summary>No sidebar</summary>

изображение

</details>

<details>

<summary>No empty space</summary>

изображение

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+mmorpg.com##.div-84937-desktop
```
Comment (Sergey-Lyapin) [inline BaseFilter/sections/specific.txt:346]: There are several such ad leftovers out there, this rule only hides one of them.

File: BaseFilter/sections/specific.txt [hunk 2]
```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
...
+mmorpg.com##.tabs--fill_last
```
Comment (Sergey-Lyapin) [inline BaseFilter/sections/specific.txt:347]: In my opinion this rule is too generic, it needs to be made more specific.

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:346]: Можно чуть получше, универсальнее заблочить. Размер 250 часто используется в рекламных баннерах.
Если посмотреть по коду, это правило скрыло бы и другие рекламные вставки, которые уже скрывает Easylist менее удачными правилами.

```suggestion
mmorpg.com##div[style^="min-height:250px;max-width:100%;"]
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Sergey-Lyapin — COMMENTED** [review]
By the way, there are 2 more ad leftovers:
ad1
ad2

File: BaseFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply BaseFilter/sections/specific.txt:347]: Maybe `.sidebar > div.tabs--fill_last` is ok

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/specific.txt [hunk 3]
```diff
@@ -343,6 +343,10 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+mmorpg.com##.single-page > div.tabs
+mmorpg.com##.sidebar > .mb-5:has(> div > div[id="div-84407"])
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:347]: Можно чуть универсвльнее. Числовое значение там скорее всего не постоянное
```suggestion
mmorpg.com##.sidebar > .mb-5:has(> div[style^="min-height:"] > div[id^="div-"])
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@vankos Please resolve conflicts.

**Review by zloyden — APPROVED** [review]

**Pull request merged**